src/src_V2/components/authInfoWrapper/index.jsx

```diff
@@ -0,0 +1,12 @@
+import styles from './styles.module.css';
+import logo from '../../assets/Travely.png';
+
+export default function AuthInfoWrapper({ ...props }) {
```

Comment:
pls destructure { children } and use it in line 9

Reply:
Thanks.